src/function/fetch.rs

```diff
@@ -15,14 +15,18 @@ where
         let (zalsa, zalsa_local) = db.zalsas();
         zalsa.unwind_if_revision_cancelled(zalsa_local);
 
+        let database_key_index = self.database_key_index(id);
+        #[cfg(debug_assertions)]
```

Comment:
This sort of defeats the purpose of tracing but I hopefully will never have to debug release-mode only salsa bugs